### PR TITLE
3.29.0

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.29.0](https://github.com/sonata-project/SonataMediaBundle/compare/3.28.0...3.29.0) - 2020-11-30
+### Added
+- [[#1863](https://github.com/sonata-project/SonataMediaBundle/pull/1863)] Comments in `GalleryAdmin.php` for the next-major translations refactoring ([@gremo](https://github.com/gremo))
+
+### Changed
+- [[#1863](https://github.com/sonata-project/SonataMediaBundle/pull/1863)] `SonataMediaBundle.de.xliff` re-arranged translation keys ([@gremo](https://github.com/gremo))
```
Comment (franmomu) [inline CHANGELOG.md:10]: Are these ones necessary?

Comment (phansys) [reply CHANGELOG.md:10]: I guess not, but this is the result of the `release` command.
Please, make a suggestion about the entries that you consider irrelevant in order to make the decision clear.

Comment (franmomu) [reply CHANGELOG.md:10]: Sorry I meant the `Added` and `Changed` from https://github.com/sonata-project/SonataMediaBundle/pull/1863, shall we just remove the comment and run the command again?

Comment (franmomu) [reply CHANGELOG.md:10]:
```suggestion
```

Comment (phansys) [reply CHANGELOG.md:10]: I other releases, where similar situations occurred, I just manually edit the output. So I think we could do the same here.

**Review by franmomu — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**